frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx

```diff
@@ -0,0 +1,67 @@
+import { json } from '@remix-run/node';
+import type { ActionFunctionArgs, LoaderFunctionArgs } from '@remix-run/node';
+import { Form, useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import { Button, ButtonLink } from '~/components/buttons';
+import { InputRadios } from '~/components/input-radios';
+import { getLookupService } from '~/services/lookup-service.server';
+import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('demographics-oral-health-questions', 'gcweb');
+
+export const handle = {
+  breadcrumbs: [
+    // prettier-ignore
+    { labelI18nKey: 'demographics-oral-health-questions:part1.breadcrumbs.canada-ca', to: 'www.google.ca' },
+  ],
```

Comment:
I noticed in the Figma prototype that the breadcrumbs for all pages are the same. If that's the case, it may be best to specify the breadcrumbs in the intake form layout instead.

Reply:
breadcrumb removed for now